src/libsync/discovery.cpp

```diff
@@ -409,7 +409,7 @@ void ProcessDirectoryJob::processFileAnalyzeRemoteInfo(
         if (serverEntry.remotePerm.isNull())
             missingData.append(tr("permissions"));
         if (serverEntry.etag.isEmpty())
-            missingData.append(tr("etag"));
+            missingData.append("ETag");
```

Comment:
Should be `QStringLiteral("ETag")`. I didn't look the exact error yet but I guess that's why the CI is failing this. ;-)

Reply:
And I was wrong, the CI didn't barf on this (although I still advise this should be done).

Reply:
Changes are done and tests are ok.